base/div.jl

```diff
@@ -123,31 +123,31 @@ julia> fld.(-5:5, 3)'
 Because `fld(x, y)` implements strictly correct floored rounding based on the true
 value of floating-point numbers, unintuitive situations can arise. For example:
 ```jldoctest
-julia> fld(6.0,0.1)
+julia> fld(6.0, 0.1)
 59.0
-julia> 6.0/0.1
+julia> 6.0 / 0.1
 60.0
-julia> 6.0/big(0.1)
+julia> 6.0 / big(0.1)
 59.99999999999999666933092612453056361837965690217069245739573412231113406246995
 ```
 What is happening here is that the true value of the floating-point number written
-as `0.1` is slightly larger than the numerical value 1/10 while `6.0` represents
-the number 6 precisely. Therefore the true value of `6.0 / 0.1` is slightly less
-than 60. When doing division, this is rounded to precisely `60.0`, but
+as `0.1` is slightly larger than the numerical value `1 / 10` while `6.0` represents
+the number `6` precisely. Therefore the true value of `6.0 / 0.1` is slightly less
```

Comment:
Backticks represent Julia code, but 1/10 and 6 are mathematical concepts here, free from any implementation details.
```suggestion
as `0.1` is slightly larger than the numerical value 1/10 while `6.0` represents
the number 6 precisely. Therefore the true value of `6.0 / 0.1` is slightly less
```

Reply:
Ha, good point, thanks for the clarification.